ext/curl/interface.c

```diff
@@ -511,73 +511,73 @@ PHP_MINIT_FUNCTION(curl)
 
 	/* */
 	REGISTER_CURL_CONSTANT(CURLE_ABORTED_BY_CALLBACK);
-	REGISTER_CURL_CONSTANT(CURLE_BAD_CALLING_ORDER);
+	REGISTER_LONG_CONSTANT("CURLE_BAD_CALLING_ORDER", 44, CONST_PERSISTENT | CONST_DEPRECATED);
```

Comment:
Where do these hardcoded values come from?

Assuming that these can never actually be returned by curl (anymore), it might make sense to map them to something that is definitely not going to be used, such as `-1` or a large number. These specific constants look fragile.